src/platform/unix.rs

```diff
@@ -5,61 +5,61 @@ use std::ptr;
 use std::sync::Arc;
 
 use {
-    EventsLoop,
+    EventLoop,
     LogicalSize,
-    MonitorId,
+    MonitorHandle,
     Window,
     WindowBuilder,
 };
-use platform::{
-    EventsLoop as LinuxEventsLoop,
+use platform_impl::{
+    EventLoop as LinuxEventLoop,
     Window as LinuxWindow,
 };
-use platform::x11::XConnection;
-use platform::x11::ffi::XVisualInfo;
+use platform_impl::x11::XConnection;
+use platform_impl::x11::ffi::XVisualInfo;
 
 // TODO: stupid hack so that glutin can do its work
 #[doc(hidden)]
-pub use platform::x11;
+pub use platform_impl::x11;
 
-pub use platform::XNotSupported;
-pub use platform::x11::util::WindowType as XWindowType;
+pub use platform_impl::XNotSupported;
+pub use platform_impl::x11::util::WindowType as XWindowType;
 
-/// Additional methods on `EventsLoop` that are specific to Linux.
-pub trait EventsLoopExt {
-    /// Builds a new `EventsLoop` that is forced to use X11.
+/// Additional methods on `EventLoop` that are specific to Linux.
```

Comment:
If the trait is named `EventLoopUnix`, then these are methods that are specific to unix-like OSes, not only Linux. I believe *BSD uses x11 too for example.